impl/.rustfmt.toml

```diff
@@ -1 +0,0 @@
-tab_spaces = 2
```

Comment:
We don't need this anymore?

Reply:
There were two `rustfmt.toml` files, The one in the root of the repo seemed to have more in it so I kept that over this one.

Reply:
SGTM